app/src/main/res/values/styles.xml

```diff
@@ -6,6 +6,14 @@
         <item name="android:navigationBarColor">@color/translucent_background</item>
     </style>
 
+    <style name="Hedvig.Theme.LoggedIn">
+        <item name="android:navigationBarColor">@color/color_background_85</item>
```

Comment:
Shouldn't the navigation bar be transparent and the views behind it inset instead?

Reply:
It does not seem possible to have the navigationbar completely transparent, if i use this color #00FFFFFF it increases the alpha as seen in this image ![bild](https://user-images.githubusercontent.com/26031482/81163240-b8fdf300-8f8e-11ea-9bb2-cc5188766c4f.jpg)

Reply:
I think it should be possible to have the navigation bar transparent, although on 3-button navigation modes it usually gets some color by default (which is fine).

Reply:
isn't it nicer looking if the navigationbar has the exact same color as the tab bar? since it is possible

Reply:
Good point. Can you attach screenshots using this color on 3-button + gesture navigation? Also using full transparent on gesture navigation 🙂

Reply:
With `<item name="android:navigationBarColor">@color/color_background_85</item>`
![Screenshot_20200507-100634_Hedvig Dev](https://user-images.githubusercontent.com/26031482/81270444-0a6bb800-904b-11ea-94fd-07dfb634fe10.jpg)

With `<item name="android:navigationBarColor">@color/color_background_85</item>`
![Screenshot_20200507-100714_Hedvig Dev](https://user-images.githubusercontent.com/26031482/81270595-4272fb00-904b-11ea-9d46-68a248a7b888.jpg)

With `<item name="android:navigationBarColor">@color/transparent</item>`
![Screenshot_20200507-100913_Hedvig Dev](https://user-images.githubusercontent.com/26031482/81270633-5159ad80-904b-11ea-9207-96ddeba1c356.jpg)

Reply:
Alright, that looks dope. Lets go with that